specification/schemas/return-reasons/ReturnReason.json

```diff
@@ -15,8 +15,17 @@
               "description": "A unique code for the return reason."
             },
             "description": {
-              "type": "string",
-              "example": "The item isn't what I expected it to be."
+              "oneOf": [
+                {
+                  "$ref": "#/components/schemas/ReturnReasonManagementDescription"
+                },
+                {
+                  "type": "string",
+                  "description": "A localized return reason description",
+                  "example": "The item isn't what I expected it to be."
+                }
```

Comment:
I would swap these `oneOf` elements, so the JSON example shows the "string" version by default. This is more clear for regular people using the endpoint (the management version is mainly for us).

Reply:
done